packages/webview_flutter/webview_flutter_android/test/android_webview_test.mocks.dart

```diff
@@ -22,8 +22,8 @@ import 'test_android_webview.pigeon.dart' as _i6;
 // ignore_for_file: unnecessary_parenthesis
 // ignore_for_file: camel_case_types
 
-class _FakeDownloadListener_0 extends _i1.Fake
-    implements _i2.DownloadListener {}
+class _FakeDownloadListener_0 extends _i1.Fake implements _i2.DownloadListener {
+}
```

Comment:
Please, revert this file, this pr has nothing to do with the webview_flutter package.